[2025-02-06T15:24:07.923+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T15:24:07.949+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-06T15:24:07.963+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-06T15:24:07.964+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-06T15:24:07.981+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_online_store_data> on 2025-02-06 00:01:00+00:00
[2025-02-06T15:24:07.996+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=254) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T15:24:07.997+0000] {standard_task_runner.py:72} INFO - Started process 258 to run task
[2025-02-06T15:24:07.999+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_online_store_data', 'scheduled__2025-02-06T00:01:00+00:00', '--job-id', '315', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpss5mj8tx']
[2025-02-06T15:24:08.002+0000] {standard_task_runner.py:105} INFO - Job 315: Subtask extract_online_store_data
[2025-02-06T15:24:08.060+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:01:00+00:00 [running]> on host f34e19f1f8c5
[2025-02-06T15:24:08.148+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_online_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:01:00+00:00'
[2025-02-06T15:24:08.150+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T15:24:08.150+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T15:24:08.151+0000] {logging_mixin.py:190} INFO - Current task name:extract_online_store_data state:running start_date:2025-02-06 15:24:07.950372+00:00
[2025-02-06T15:24:08.151+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-06T15:24:08.151+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T15:24:08.174+0000] {base.py:84} INFO - Retrieving connection 'online_store_conn'
[2025-02-06T15:24:08.178+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM online_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-06T15:24:08.190+0000] {sql.py:562} INFO - Rows affected: 5077
[2025-02-06T15:24:08.314+0000] {etl.py:150} INFO - Extracted orders 5077 new rows from db_online_store.
[2025-02-06T15:24:08.319+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM customers 
            WHERE updated_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-06T15:24:08.323+0000] {sql.py:562} INFO - Rows affected: 2999
[2025-02-06T15:24:08.363+0000] {etl.py:170} INFO - Extracted customers 2999 new rows from db_online_store.
[2025-02-06T15:24:08.367+0000] {python.py:240} INFO - Done. Returned value was: ('/tmp/online_oders_data.json', '/tmp/online_customer_data.json')
[2025-02-06T15:24:08.392+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T15:24:08.392+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_online_store_data, run_id=scheduled__2025-02-06T00:01:00+00:00, execution_date=20250206T000100, start_date=20250206T152407, end_date=20250206T152408
[2025-02-06T15:24:08.403+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T15:24:08.404+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T15:24:08.404+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-06 15:24:02.325020+00:00
[2025-02-06T15:24:08.404+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-06T15:24:08.453+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T15:24:08.485+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T15:24:08.487+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-07T11:21:20.264+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T11:21:20.292+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-07T11:21:20.305+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:01:00+00:00 [queued]>
[2025-02-07T11:21:20.305+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-07T11:21:20.324+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_online_store_data> on 2025-02-06 00:01:00+00:00
[2025-02-07T11:21:20.344+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=97) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T11:21:20.345+0000] {standard_task_runner.py:72} INFO - Started process 100 to run task
[2025-02-07T11:21:20.346+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_online_store_data', 'scheduled__2025-02-06T00:01:00+00:00', '--job-id', '611', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpp2xrffk2']
[2025-02-07T11:21:20.348+0000] {standard_task_runner.py:105} INFO - Job 611: Subtask extract_online_store_data
[2025-02-07T11:21:20.408+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:01:00+00:00 [running]> on host f34e19f1f8c5
[2025-02-07T11:21:20.559+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_online_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:01:00+00:00'
[2025-02-07T11:21:20.560+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T11:21:20.561+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T11:21:20.561+0000] {logging_mixin.py:190} INFO - Current task name:extract_online_store_data state:running start_date:2025-02-07 11:21:20.293244+00:00
[2025-02-07T11:21:20.561+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-07T11:21:20.562+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T11:21:20.587+0000] {base.py:84} INFO - Retrieving connection 'online_store_conn'
[2025-02-07T11:21:20.593+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM online_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-07T11:21:20.618+0000] {sql.py:562} INFO - Rows affected: 9478
[2025-02-07T11:21:20.992+0000] {etl.py:150} INFO - Extracted orders 9478 new rows from db_online_store.
[2025-02-07T11:21:20.999+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM customers 
            WHERE updated_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-07T11:21:21.006+0000] {sql.py:562} INFO - Rows affected: 3602
[2025-02-07T11:21:21.080+0000] {etl.py:170} INFO - Extracted customers 3602 new rows from db_online_store.
[2025-02-07T11:21:21.085+0000] {python.py:240} INFO - Done. Returned value was: ('/tmp/online_oders_data.json', '/tmp/online_customer_data.json')
[2025-02-07T11:21:21.140+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T11:21:21.142+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_online_store_data, run_id=scheduled__2025-02-06T00:01:00+00:00, execution_date=20250206T000100, start_date=20250207T112120, end_date=20250207T112121
[2025-02-07T11:21:21.152+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T11:21:21.153+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T11:21:21.153+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-07 11:21:14.687092+00:00
[2025-02-07T11:21:21.153+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-07T11:21:21.210+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T11:21:21.252+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-07T11:21:21.254+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
